charts/librephotos/templates/deploymentBackend.yaml

```
@@ -82,8 +82,13 @@ spec:
           envFrom:
             - secretRef:
                 name: "{{ $resourceName }}-secrets"
+          {{- if .Values.existingSecret }}
```

```suggestion
            - secretRef:
               name: {{ if .Values.existingSecret -}} {{ .Values.existingSecret | quote }} {{- else }} "{{ $resourceName }}-creds" {{- end }}
```

something like this will eliminate the lines